cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,137 @@
+    | ap-southeast-1 (Singapore) | [![Launch Stack](https://cdn.rawgit.com/buildkite/cloudformation-launch-stack-button-svg/master/launch-stack.svg)](https://ap-southeast-1.console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=startChatContactAPI&templateURL=https://s3-ap-southeast-1.amazonaws.com/ap-southeast-1.start-chat-contact-api-cfn/cloudformation.yaml) |
+
+
+4. Update the CloudFormation template with details from your connect instance.
+    - Fetch the Contact ID and Instance ID from your Connect instance by going to your contact flows and selecting **Sample queue customer** and clicking on **Show additional flow information** on top left. 
+    - `contactFlowId` \ `instanceId`: You can find the contact flow id and instance id when viewing a contact flow. For example, if the arn for your flow is 'arn:aws:connect:us-west-2:123456789012:instance/11111111-1111-1111-1111-111111111111/contact-flow/22222222-2222-2222-2222-222222222222', the contact flow id is '22222222-2222-2222-2222-222222222222' and instance id is 11111111-1111-1111-1111-111111111111
+
+        ![Contact Flow Id's](images/updateContactFlowIds.png)
+
+    - Go to your [CloudFormation stack](https://us-west-2.console.aws.amazon.com/cloudformation/home) by clicking on Stacks and then startChatContactAPI on the left hand side. Update the Contact Flow ID and the Instance ID in your cloud formation stack
+    ![Update Cloud Formation Stack](images/updateCloudFormationStack.png)
+    -   These are added as Lambda environment variables.
 
-Note: you will want to show the widget only when there is a conversation in progress. If the widget is visible when there is no conversation, you will just see a loading spinner.
+## Getting started with Pre Built Widget
 
-1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo.
+You can refer to the `widgetIndex.html` file in this repo to see an example of how to use the widget.
 
+1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo. Also import jQuery
     ```html
-    <script src="amazon-connect-chat-interface.js"></script>
+        <script src="amazon-connect-chat-interface.js"></script>
+        <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
     ```
 
-2. Initialize the Chat Interface on page load. Note: you need to update this to include the root id of the div where the customer chat widget will live.
-
-    ```js
-    $(document).ready((a) => {
-      connect.ChatInterface.init({
-        containerId: 'root' // This is the id of the container where you want the widget to reside
-        });
-    });
+2. Create a `div` with id of `root`. 
+    ```html
+    <div id="root">
```

Comment:
you changed this to root (it should be chat)
but you didn't change the code.

Reply:
Changed it. Good catch